pkg/cmd/ssh_alicloud.go

```diff
@@ -198,9 +199,9 @@ func (a *AliyunInstanceAttribute) createBastionHostSecurityGroup() {
 			os.Exit(2)
 		}
 		fmt.Println("Configuring bastion host security group rules...")
-		_, err = ExecCmdReturnOutput("bash", "-c", "aliyun ecs AuthorizeSecurityGroup --Policy Accept --NicType intranet --Priority 1 --SourceCidrIp 0.0.0.0/0 --PortRange 22/22 --IpProtocol udp --SecurityGroupId="+a.BastionSecurityGroupID)
+		_, err = ExecCmdReturnOutput("bash", "-c", "aliyun ecs AuthorizeSecurityGroup --Policy Accept --NicType intranet --Priority 1 --SourceCidrIp %s --PortRange 22/22 --IpProtocol udp --SecurityGroupId="+a.BastionSecurityGroupID, a.MyPublicIP)
```

Comment:
I think it will be better to extract this command in function which accepts `protocol`. 
It is difficult to see now where is the difference between these two `bash` commands

Reply:
will remove udp it seem SSH protocol no needed